### DISCUSSION TITLE
Why does Supabase have to break on me every two weeks

### DISCUSSION THREAD

**Discussion by tonyborchert** [supabase/supabase#4214] [Questions] [answered]
Why can't before a update I be asked if I want to update? Like not even the dart client library is coming behind and im having to depend on some weird branch to get it fixed until the PR finally merges

**Comment by inian** [comment, marked as answer]
We will try to do better here @bit-burger. We have started publishing release notes with notable changes [here](https://github.com/supabase/supabase/releases). 

What was the exact breaking change that had to be fixed in the dart library?

**Reply by tonyborchert** [reply]
thanks for the release notes and notable changes, haven't seen that so thanks for sharing

**Reply by tonyborchert** [reply]
But this for example is still not fixed: https://github.com/supabase-community/realtime-dart/pull/34#issue-1079989958, i've been waiting ages for a merge (not my pull request)

**Reply by inian** [reply]
Sorry about the delay in fixing this, the PR has been merged. In the future, we will prioritise merging these breaking changes if any in our community libraries
